Makefile

```
@@ -71,10 +86,39 @@ build-updater:
+	CODEX_FINAL_APP_DIR="$(APP_DIR)" \
+		./scripts/rebuild-candidate.sh --install "$(DMG)"
+
+inspect-upstream:
+	@echo "[make] Inspecting upstream DMG"
+	./install.sh --inspect --report-dir "$(REBUILD_REPORT_DIR)" "$(DMG)"
+
 build-app:
 	@echo "[make] Regenerating codex-app from DMG"
 	./install.sh "$(DMG)"
 
+rebuild-next:
```

checkmake (0.3.2)

[warning] Target body for "rebuild-next" exceeds allowed length of 5 lines (8).

(maxbodylength)